hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+
+`ExtCodeHash`
+
+**Ethereum:** Hash of the chosen account's code, the empty hash (0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470) if the account has no code, or 0 if the account does not exist or has been destroyed.
+
+**Hedera:** For addresses in range 0x0 → 0x3E8 (0.0.0 → 0.0.1000) it returns the empty hash.
+
+### Balance operation
+
+**Ethereum:** Returns balance or 0 if an account or contract doesn’t exist on the address. This is valid for any address.
+
+**Hedera:**
+1. For addresses in range 0x0 → 0x2EE (0.0.0 → 0.0.750): returns 0 as if account doesn’t exist.
+2. For addresses in range 0x2EF → 0x3E8 (0.0.751 → 0.0.1000): returns balance or 0 if an account or contract doesn’t exist on the address. This behavior is the same for addresses above 0.0.1000.
+
+| Address                            | in Ethereum                                                                   | in Hedera                                                                     |
+|------------------------------------|-------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| 0x0                                | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x1 → 0x9                          | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0xA → 0x166                        | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x167 (0.0.359)                    | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x168, 0x169 (0.0.360, 0.0.361)    | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x16a → 0x2EE (0.0.361 → 0.0.750)  | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns 0 as if account doesn’t exist                                       |
+| 0x2EF → 0x3E8 (0.0.751 → 0.0.1000) | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns balance or 0 if an account or contract doesn’t exist on the address |
+| 0x3E8 → ♾️                         | ✅ returns balance or 0 if an account or contract doesn’t exist on the address | ✅ returns balance or 0 if an account or contract doesn’t exist on the address |
+
+### SelfDestruct operation
+
+**Ethereum:** success for any address. If the beneficiary is the same as contract address, it burns the eth and destructs the contract.
```

Comment:
re "destructs the contract" - no longer true post Cancun (which isn't deployed yet, admittedly, but will be soon) - Cancun semantics for self destruct will apply: Sweep (if possible, otherwise fail as the notes below show) then self destruct _only if created in same transaction_)

Reply:
It seems that the Cancun upgrade went live on March 13 so I will update the documentation according to your comment. Thanks for pointing that out.